packages/blocks/src/note-block/note-edgeless-block.ts

```diff
@@ -490,20 +488,15 @@ export class EdgelessNoteBlockComponent extends toGfxBlockComponent(
           .model=${this.model}
           .display=${!this._editing}
           .host=${this.host}
-          .zoom=${this.rootService?.viewport.zoom ?? 1}
+          .zoom=${this.gfx.viewport.zoom ?? 1}
           .editing=${this._editing}
         ></edgeless-note-mask>
       </div>
     `;
   }
 
-  override toZIndex() {
-    // FIXME: weird empty rootService
-    return this.rootService?.layer.getZIndex(this.model).toString() ?? '0';
-  }
-
-  override get rootService() {
-    return super.rootService as EdgelessRootService;
+  get rootService() {
```

Comment:
FYI, We're aiming to remove the block services.

Reply:
What’s the replacement? I also need to do some cleanup tasks in edgeless during the next iteration.

Reply:
Just use `this.std.get` directly to get the extension you want.